R/WebApi.R

```diff
@@ -136,6 +136,8 @@ getCdmSources <- function(baseUrl) {
     }
     data.frame(sourceName = s$sourceName,
                sourceKey = s$sourceKey,
+               sourceNameShort = gsub("\\s*\\([^\\)]+\\)","",as.character(s$sourceName)),
```

Comment:
@gowthamrao -- I don't think these fields necessarily provide value for sites outside of Janssen. For instance, only at Janssen is there a source version (load id) in the name. 

I think we should leave these out, as they are not going to be helpful for other sites, and may instead cause confusion.